package.json

```diff
@@ -23,11 +23,34 @@
   },
   "eslintConfig": {
     "root": true,
-    "extends": "@hmudesign",
+    "extends": [
```

Comment:
These linter config changes are more or less just pulling the base config out of @hmudesign.  Using two different versions of @typescript-eslint/eslint-plugin (one called from hmudesign, the other from react-scripts) seemed to cause issues with starting up the linter for the dev server.